pkg/printers/internalversion/printers.go

```diff
@@ -45,6 +46,8 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/util/duration"
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apiserver/pkg/authentication/user"
+	// "k8s.io/apiserver/pkg/util/apihelpers"
```

Comment:
can we remove this instead of commenting?

Reply:
oops, yes.  Next rev.